plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/events_log_analyzer.py

```diff
@@ -152,3 +152,15 @@ def plot_link_up_down_count_per_aggregation_time(self):
         self._save_pivot_data_in_bars(
             pivot_links_data, "Time", "Number of Events", "Link up/down events", "Event"
         )
+
+    def full_analysis(self):
+        images, _, _ = super().full_analysis()
+        critical_events_headers = ["timestamp", "event_type", "event", "count"]
+        lists = [
+            (
+                self.get_critical_event_bursts(),
```

Comment:
get_critical_event_bursts returns list only since it was implemented before the PDF dataframe support.
If you think you can do it now, you can change it to return a data frame. If not, let me know and I'll open a ticket to fix it when we can

Reply:
done